build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
On Linux, we expect to get libclrjit.so. You need to use Ubuntu build of CoreCLR to crossgen assemblies for Ubuntu - you cannot use Windows binaries to do that.

Here is what I am thinking:

1) On a given Platform (Windows OR OSX OR RHEL OR Ubuntu), build the product and pass a switch "-restorefxfortest".
2) When build.sh/bat sees "-restorefxfortest", it will restore the packages required for generating the Core_Root for that platform, use the crossgen from step (1) to compile them and you will get core_Root for running tests.
3) Send Core_Root to Helix - nothing needs to build on Helix.

What do you think? Will this not work?